packages/common/src/store/ui/stripe-modal/sagas.ts

```diff
@@ -34,13 +34,13 @@ function* handleStripeSessionChanged({
     if (onrampSucceeded) {
       yield* put(onrampSucceeded)
     }
-    yield* put(setVisibility({ modal: 'StripeOnRamp', visible: false }))
+    yield* put(setVisibility({ modal: 'StripeOnRamp', visible: 'closing' }))
   }
 }
 
 function* handleCancelStripeOnramp() {
   const { onrampCanceled } = yield* select(getStripeModalState)
-  yield* put(setVisibility({ modal: 'StripeOnRamp', visible: false }))
+  yield* put(setVisibility({ modal: 'StripeOnRamp', visible: 'closing' }))
```

Comment:
Is this from the recent modal changes @rickyrombo did? Or what motivated changing this to a string?

Reply:
It's not related to my change. Ideally, we wouldn't set this manually like this but have some hook generate the `'closing'` vs `false` for you, and you don't have to see or understand the difference. `'closing'` is distinct from `false` in that `'closing'` means that it's in the _process_ of closing (animating closed) and `false` is that it's fully disappeared. This allowed @dylanjeffers to do [conditional rendering] of Modals to improve perf.